examples/network-manager-app/linux/main.cpp

```diff
@@ -33,6 +36,13 @@ ByteSpan ByteSpanFromCharSpan(CharSpan span)
     return ByteSpan(Uint8::from_const_char(span.data()), span.size());
 }
 
+std::optional<DefaultThreadNetworkDirectoryServer> gThreadNetworkDirectoryServer;
+void emberAfThreadNetworkDirectoryClusterServerInitCallback(chip::EndpointId endpoint)
```

Comment:
The "ClusterServerInitCallback" bit is for the cluster impl itself to use.  App init should happen in `emberAfThreadNetworkDirectoryClusterInitCallback` (note: no "Server" in there).